business/workloads.go

```diff
@@ -326,7 +328,13 @@ func (in *WorkloadService) GetWorkload(ctx context.Context, criteria WorkloadCri
 		return nil, err
 	}
 
-	workload, err2 := fetchWorkload(ctx, in.businessLayer, criteria)
+	cluster := criteria.Cluster
+	if cluster == "" {
```

Comment:
If we don't send any parameter at all (http://localhost:3000/console/namespaces/bookinfo/workloads/details-v1?duration=60&refresh=0), I get a nullPointer. It looks like this is converted to "null": 

![image](https://user-images.githubusercontent.com/49480155/228772526-de6e6bd9-da3f-44a8-8f41-dfe04bf1f681.png)

Reply:
thanks for the catch Josune, I really missed that